cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
```

Comment:
```suggestion
import "@testing-library/cypress/add-commands";

```

Please add `@testing-library/cypress`, just like what we did here https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/cypress/support/index.js#L16.